site/ko/beta/tutorials/generative/dcgan.ipynb

```diff
@@ -0,0 +1,988 @@
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "NS2GWywBbAWo"
+   },
+   "outputs": [],
+   "source": [
+    "EPOCHS = 50\n",
+    "noise_dim = 100\n",
+    "num_examples_to_generate = 16\n",
+    "\n",
+    "# 우리는 이 시드를 시간이 지나도 재활용하겠습니다. \n",
+    "# (GIF 애니메이션에서 진전을 시각화하는데에 쉽기 때문입니다.) \n",
+    "seed = tf.random.normal([num_examples_to_generate, noise_dim])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "jylSonrqSWfi"
+   },
+   "source": [
+    "훈련 루프는 생성자가 입력으로 랜덤시드를 받는 것으로 부터 시작됩니다. 그 시드값은 이미지를 생성하는데 사용되어 집니다. 감별자는 (훈련 세트에서 갖고온) 진짜 이미지와 (생성자가 생성해낸) 가짜이미지를 분류하기 위해 사용되어집니다. 각각의 모델들의 손실이 계산되어지고, 경사 (gradients)는 생성자와 감별자를 업데이트하기 위해 사용되어집니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "3t5ibNo05jCB"
+   },
+   "outputs": [],
+   "source": [
+    "# `tf.function`이 어떻게 사용되어지는지 주목해 주세요.\n",
+    "# 이 주석은 함수가 \"컴파일\"되어 지도록 합니다.\n",
+    "@tf.function\n",
+    "def train_step(images):\n",
+    "    noise = tf.random.normal([BATCH_SIZE, noise_dim])\n",
+    "\n",
+    "    with tf.GradientTape() as gen_tape, tf.GradientTape() as disc_tape:\n",
+    "      generated_images = generator(noise, training=True)\n",
+    "\n",
+    "      real_output = discriminator(images, training=True)\n",
+    "      fake_output = discriminator(generated_images, training=True)\n",
+    "\n",
+    "      gen_loss = generator_loss(fake_output)\n",
+    "      disc_loss = discriminator_loss(real_output, fake_output)\n",
+    "\n",
+    "    gradients_of_generator = gen_tape.gradient(gen_loss, generator.trainable_variables)\n",
+    "    gradients_of_discriminator = disc_tape.gradient(disc_loss, discriminator.trainable_variables)\n",
+    "\n",
+    "    generator_optimizer.apply_gradients(zip(gradients_of_generator, generator.trainable_variables))\n",
+    "    discriminator_optimizer.apply_gradients(zip(gradients_of_discriminator, discriminator.trainable_variables))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "2M7LmLtGEMQJ"
+   },
+   "outputs": [],
+   "source": [
+    "def train(dataset, epochs):\n",
+    "  for epoch in range(epochs):\n",
+    "    start = time.time()\n",
+    "\n",
+    "    for image_batch in dataset:\n",
+    "      train_step(image_batch)\n",
+    "\n",
+    "    # GIF를 위한 이미지를 바로 생성합니다.\n",
+    "    display.clear_output(wait=True)\n",
+    "    generate_and_save_images(generator,\n",
+    "                             epoch + 1,\n",
+    "                             seed)\n",
+    "\n",
+    "    # 15 에폭이 지날 때마다 모델을 저장합니다.\n",
+    "    if (epoch + 1) % 15 == 0:\n",
+    "      checkpoint.save(file_prefix = checkpoint_prefix)\n",
+    "\n",
+    "    print ('Time for epoch {} is {} sec'.format(epoch + 1, time.time()-start))\n",
```

Comment:
Please translate print function :)

Reply:
I have added the below line for the translation of the print function, but I am not sure if this is the best expression.
# 출력: 에포크 (epoch+1)에서 걸린 시간은 (현재 시간(time.time())-start)초 입니다.

Reply:
I suggest,
print (' 에포크 {} 에서 걸린 시간은 {} 초 입니다'.format(epoch +1, time.time()-start))\n",
because python print() is actually functioning programming code.